### ISSUE TITLE
Java, Maven and Osmosis recommended version

### ISSUE THREAD

**Issue by Jrmwelli** [eqasim-org/ile-de-france#251]
Hi guys, I'm trying to run the simulation for Ile-de-France but I'm facing some problems
I'm using Java 11.0.24, Maven 3.8.8 and Osmosis 0.47.1; nevertheless Maven plugins seem to require Java 17 (if I've understood well the error message)
I was facing a lot of problems earlier because I was using the newest version of osmosis (Java 11 and Osmosis 0.49.2 can't work together right now), so if someone has a combo of version that can run the latest version of this pipeline, I'm really interested
Thanks !

**Comment by sebhoerl** [comment]
You're right, we should probably revise the documentation to find the right version. The continuous integration tests currently use:
- Java 17.0.11
- Maven 3.8.8
- osmosis 0.48.2
